web/figures.py

```diff
@@ -93,21 +94,28 @@ def get_figures(trips: Trips, charging: Tuple[dict]):
         data=[tr.get_info() for tr in trips[::-1]],
         page_size=50
     )
-    # consumption_fig
-    consumption_df = DataFrame.from_records(trips.get_long_trips())
-    consumption_fig = px.line(consumption_df, x="date", y="consumption", title='Consumption of the car')
-    consumption_fig.update_layout(yaxis_title="Consumption kWh/100Km")
+    # defaults for empty trips, based on official Corsa-e c
+    kw_per_km = 17.4
```

Comment:
shouldn't the default consumption be based on the actual vehicle instead of just one model?